### DISCUSSION TITLE
How to Responsibly Drop IE11 Support

### DISCUSSION THREAD

**Discussion by nakleiderer** [nakleiderer/kleiderer-site#718] [Articles]
What are your thoughts on how to responsibly drop IE11 support?

First, [read the article](http://kleiderer.com/article/20200125-how-to-responsibly-drop-ie11-support), then share your thoughts below!

**Comment by spurgeonca** [comment]
> The point of these messages is to overcome unconscious ignorance. Your users should be aware that they are missing out on a better experience and let them make a conscious choice to upgrade or stay with their current browser. Ideally, these messages will accelerate your users’ adoption of a modern browser, resulting in lower business risk for dropping legacy browser support.

What I appreciate about this article is the foundational truth that your code exists to serve people. If a user can't access what you've created, they are at the loss. **Find** ways to help them experience what you've worked so hard to present. **Serve** them well by educating and guiding them. Don't be at opposition with the user. They **want** to experience code, but they need a friendly, trusted guide to let them know they'll be safe and _their_ interests (not yours) is **your** number one priority.  #codehope
